### DISCUSSION TITLE
Help Error code (Pillow)

### DISCUSSION THREAD

**Discussion by kaijarahk** [ParthJadhav/Tkinter-Designer#392] [Q&A]
Hi im trying to do pip install tkdesigner and this is the error message:

<string>:46: RuntimeWarning: Pillow 8.4.0 does not support Python 3.12 and does not provide prebuilt Windows binaries. We do not recommend building from source on Windows.
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for Pillow
Failed to build Pillow
ERROR: Could not build wheels for Pillow, which is required to install pyproject.toml-based projects

**Comment by AakashJayant** [comment]
its same with me

**Reply by Huy4648** [reply]
It is same error with me.


Building wheels for collected packages: Pillow
  Building wheel for Pillow (setup.py) ... error
  error: subprocess-exited-with-error

  × python setup.py bdist_wheel did not run successfully.
  │ exit code: 1
  ╰─> [187 lines of output]

Please helo

**Comment by johnmcd88** [comment]
I'm seeing the same thing while trying to install tkdesigner on Windows 10 using 'pip install tkdesigner':

<string>:46: RuntimeWarning: Pillow 8.4.0 does not support Python 3.12 and does not provide prebuilt Windows binaries. We do not recommend building from source on Windows.

**Comment by TintedMage** [comment]
I recently started learning python, so I don't know much, but installing via pip download's and used cached version of pillow 8.4.0 which doesn't support python 11 or above ... so i installed it via source .. change the pillow version to "^10.1" in pyproject.toml
